docs/development_setup.md

```diff
@@ -3,7 +3,7 @@
 To run all components (OQT/API, Database, Website) in Docker containers simply run:
 
 ```bash
-docker-compose -f docker-compose.development.yml up -d
+docker compose up --detach
```

Comment:
Why are you switching to the long version instead of the commonly used abbreviation `-d`?

Reply:
I thought it is easier to understand for people not very familiar with docker. For copy and paste it does not make a different. People just taking it as a reference will probably know how to abbreviate it.